install/index.php

```diff
@@ -1,6 +1,10 @@
 <?php
+/*
+Some Code is powered by Tecflare Corporation
+Special Thanks to Tecflare Corporation
+*/
 set_time_limit(0);
-//error_reporting(0); //don't show errors
+error_reporting(E_ERROR); //don't show errors

```

Comment:
Error reporting should be turn off completely,otherwise this will led to higher infomation leak risk